### PR TITLE
Edgencio | MPOC-171 validation between HF, Province and District under Teste de HIV

### PR DESCRIPTION
**Pull request by Edgencio**

### CODE REVIEW

**Comment by nthfloor** [comment]
@Edgencio please resolve the merge conflicts on this PR.

File: ui/app/registration/directives/healthFacilityAddressFields.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
+'use strict';
+angular.module('bahmni.registration')
```
Comment (nthfloor) [inline ui/app/registration/directives/healthFacilityAddressFields.js:2]: @Edgencio have you included unit tests for these new functions and directive?

Comment (Edgencio) [reply ui/app/registration/directives/healthFacilityAddressFields.js:2]: Nop... i'll start working on that

Comment (nthfloor) [reply ui/app/registration/directives/healthFacilityAddressFields.js:2]: Thanks, maybe before you get too stuck in - 

@joaomachiana I recommend that we include unit tests with our changes it is good so we make sure we are producing robust and reliable code, but it also adds some extra time to the development. 

So we can either include now, which I think is best, and take a bit more time, or we start creating technical debt tickets so we can come back after and add the tests. What do you guys think?

**Pull request merged**